test/tests/no-root-password/container.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+[ "$(grep root /etc/shadow | cut -d: -f2)" = '*' ]
```

Comment:
I think `awk` would probably be more well-suited for parsing `/etc/shadow`; something like `awk -F ':' '$1 == "root" { print $2 }' /etc/shadow`, right?